test/integration/push_int_test.go

```diff
@@ -264,24 +264,6 @@ func (suite *PushIntegrationTestSuite) TestPush_NoChanges() {
 	}
 }
 
-func (suite *PushIntegrationTestSuite) TestPush_NoCommit() {
```

Comment:
Isn't this still testing a valid error case? We could just update the test to reflect the changes in this PR.